docs/conf.py

```diff
@@ -28,14 +28,14 @@
 # -- Project information ------------------------------------------------------
 project = configuration["name"]
 author = f"{configuration['authors'][0]['name']} <{configuration['authors'][0]['email']}>"
-copyright = f"{datetime.datetime.now().year}, {configuration['authors'][0]['name']}"
+copyright = f"{datetime.datetime.now().year}, {author}"
 
 release = distribution(configuration["name"]).version
 # for example take major/minor
 version = ".".join(release.split(".")[:2])
 
 # -- Options for HTML output ---------------------------------------------------
-html_title = f"{project} v{release}"
+html_title = f"{project} v{version}"
```

Comment:
```suggestion
html_title = f"{project} v{release}"
```
My vote is for keeping the full version with the patch version (`4.0.0`) instead of truncating it to major/minor (`4.0`).

Reply:
Easily done.